### PR TITLE
567 robot search timings

### LINKED ISSUE #567: Test for search and filter by content type fails randomly
The search test case "Allan can search and filter by content type" sometimes fails randomly, As an example: 

http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/486/consoleFull#1659009341d7588e9e-566a-4586-b220-5d422b09bef9

but this has happened a few more times this sprint week on other pull requests that are completely unrelated to search.  Probably a timing that has to be fixed somewhere.

### PR DESCRIPTION
**Pull request by fredvd**
Fixes #567.

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by mattss** [comment]
Test this please jenkins

File: src/ploneintranet/suite/tests/acceptance/search.robot [hunk 1]
```diff
@@ -80,11 +80,6 @@ I can follow the search result ${SEARCH_RESULT_TITLE}
     Click Link  link=${SEARCH_RESULT_TITLE}
     Page should contain  ${SEARCH_RESULT_TITLE}
 
-I can exclude content of type ${CONTENT_TYPE}
-    Unselect Checkbox  css=input[type="checkbox"][value="${CONTENT_TYPE}"]
-    Select From List By Value  css=select[name="created"]  today
-    Wait Until Element is Visible  css=dl.search-results[data-search-string*="created=today"]
-
```
Comment (mattss) [inline src/ploneintranet/suite/tests/acceptance/search.robot:87]: Can I ask why this keyword was removed? It now means that the main robot test doesn't read like a user story. Abstracting the actual interactions into a keyword means that when the UI implementation changes, the keywords can be updated and the test cases can stay the same.

**Comment by gyst** [comment]
@fredvd I agree with Matt. Can you please factor out the specifics into keywords in lib, then call those keywords?

**Comment by mattss** [comment]
They keywords were already in place, so this should just be a case of reverting the 'simplify' commit.

**Comment by fredvd** [comment]
@matts I remove the keyword because the extra filtering on created today items is what's probably causing the random failures and is not part of the test description. That left only one statement but I didn't know it isn't allowed to to explicit selects etc. in the main test descriptions. I've cleaned up the first keyword and added an extra line that checks for 'does not contain'. There's a similar new builtin keyword in selenium 1.7, but we're using 1.6 afaik.

**Comment by mattss** [comment]
@fredvd Looks good - thanks!

**Comment by mattss** [comment]
Test this please jenkins

**Pull request merged**